.travis.yml

```diff
@@ -1,12 +1,22 @@
 --- 
 services: 
   - docker
-before_install: 
+before_script: 
   - "export PYTHONPATH=$PYTHONPATH:$(pwd)"
   - "docker login -u $DOCKER_USERNAME -p $DOCKER_PASSWORD"
-  - "docker --version"
-  - "docker build --target test --tag todo-app:test ."
-
-script: 
-  - "echo \"yes\""
-  - "docker run -it --rm --env TRELLO_TOKEN=$TRELLO_TOKEN --env TRELLO_KEY=$TRELLO_KEY --env FLASK_ENV=test todo-app:test tests"
\ No newline at end of file
+  - docker login -u=_ -p=$HEROKU_API_KEY registry.heroku.com
+  - docker pull cjarvis2012/todo-app
+script:
+  - "docker build --target test --tag todo-app:test --cache-from cjarvis2012/todo-app ."
+  - "docker run -it --rm --env TRELLO_TOKEN=$TRELLO_TOKEN --env TRELLO_KEY=$TRELLO_KEY --env FLASK_ENV=test todo-app:test tests"
```

Comment:
Is there any reason you're running this with -`it`? It shouldn't need to be run interactively